pkg/virt-api/webhooks/mutating-webhook/mutators/vm-mutator.go

```diff
@@ -61,6 +63,13 @@ func (mutator *VMsMutator) Mutate(ar *admissionv1.AdmissionReview) *admissionv1.
 	// Set VM defaults
 	log.Log.Object(&vm).V(4).Info("Apply defaults")
 	mutator.setDefaultMachineType(&vm)
+
+	if err = mutator.setPreferredInstancetype(&vm); err != nil {
+		log.Log.Reason(err).Error("Unable to set preferred instancetype")
+	}
+	if err = mutator.setPreferredPreference(&vm); err != nil {
+		log.Log.Reason(err).Error("Unable to set preferred preference")
+	}
```

Comment:
these errors are effectively ignored by the mutator. they're logged, but the VM's acceptance isn't impacted by them

Reply:
Yeah indeed and this was by design at the time, I still think there's a case for ignoring failures here otherwise we might end up duplicating volumes (DVs not PVCs) with and without annotations.